### PR TITLE
Add missing workflows

### LINKED ISSUE #13: 2.4.0 milestone closed but not yet released
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 2.4.0

#### Summary

I see the [milestone 2.4.0](https://github.com/mezzio/mezzio-laminasviewrenderer/milestone/5) was closed in october. I assume this means that an automatic release should have happened back then. However I don't see any 2.4.0 release. The [most recent release is 2.3.0](https://github.com/mezzio/mezzio-laminasviewrenderer/releases).

Is it a mishap of automatic release ? or is it intentional ?

#### Current behavior

2.4.0 is not released

#### Expected behavior

Have 2.4.0 released

### PR DESCRIPTION
**Pull request by PowerKiKi**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description
Copied straight from mezzio/mezzio

Fixes #13

### CODE REVIEW

**Comment by PowerKiKi** [comment]
Of course I am not able to set up secrets and such, but I assume they are already set on the org level.

File: .github/workflows/docs-build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: docs-build
+
```
Comment (Ocramius) [inline .github/workflows/docs-build.yml:2]: @froschdesign can you check if this workflow is OK as-is, or if more is needed?

**Review by Ocramius — APPROVED** [review]

File: .github/workflows/docs-build.yml (hunk 1 above)
Comment (froschdesign) [reply .github/workflows/docs-build.yml:2]: @Ocramius 
Looks good! 👍 

_Sorry for the delay, too many notifications here on GitHub._

**Comment by Slamdunk** [comment]
@froschdesign @Ocramius https://github.com/mezzio/mezzio-laminasviewrenderer/pull/12 works fine in my prod and this looks approved: can we release `2.4.0` minor?

**Comment by Ocramius** [comment]
@Slamdunk @PowerKiKi done, thanks!

**Comment by Slamdunk** [comment]
@Ocramius you're awesome, wait me in Apex so you can earn few easy RP

**Pull request merged**